README.md

```diff
@@ -304,6 +304,7 @@ Approvers ([@open-telemetry/java-approvers](https://github.com/orgs/open-telemet
 
 - [Jason Plumb](https://github.com/breedx-splk), Splunk
 - [Josh Suereth](https://github.com/jsuereth), Google
```

Comment:
I wonder if Josh should still be on this list...

Reply:
Emeritus probably.